config/metrics_endpoints.json

```diff
@@ -1,635 +1,1111 @@
+        "academy": "Object Storage",
+        "readthedocs": "",
+        "helpdesk": "Object Storage",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "coverll": "renarsvilnis/fiware-object-storage-ge",
+        "github": [
+            "renarsvilnis/fiware-object-storage-ge"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "OpenMTC - oneM2M",
+        "catalogue": "openmtc",
+        "academy": "",
+        "readthedocs": "fiware-openmtc",
+        "helpdesk": "OpenMTC",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "OpenMTC/OpenMTC"
+        ],
+        "docker": [
+            "openmtc/influxdb-app-arm",
+            "openmtc/influxdb-app-amd64",
+            "openmtc/cul868-ipe-arm",
+            "openmtc/cul868-ipe-amd64",
+            "openmtc/orion-context-broker-app-arm",
+            "openmtc/orion-context-broker-app-amd64",
+            "openmtc/gateway-arm",
+            "openmtc/gateway-amd64",
+            "openmtc/backend-arm",
+            "openmtc/backend-amd64",
+            "openmtc/mqtt-connector-app-arm",
+            "openmtc/mqtt-connector-app-amd64",
+            "openmtc/csv-injector-app-arm",
+            "openmtc/csv-injector-app-amd64"
+        ]
+    },
+    {
+        "enabler": "Orion - Context Broker",
+        "catalogue": "orion-context-broker",
+        "academy": "Orion Context Broker",
+        "readthedocs": "fiware-orion",
+        "helpdesk": "Orion",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/fiware-orion"
+        ],
+        "docker": [
+            "fiware/orion"
+        ]
+    },
+    {
+        "enabler": "POI Data Provider",
+        "catalogue": "poi-data-provider",
+        "academy": "POI Data Provider",
+        "readthedocs": "fiware-poidataprovider",
+        "helpdesk": "POI Provider",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Chiru/FIWARE-POIDataProvider"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "Pegasus - PaaS Manager",
+        "catalogue": "paas-manager-pegasus",
+        "academy": "Pegasus",
+        "readthedocs": "fiware-paas",
+        "helpdesk": "Pegasus",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "coverll": "telefonicaid/fiware-paas",
+        "github": [
+            "telefonicaid/fiware-paas"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "Perseo",
+        "catalogue": "kurento",
+        "academy": "",
+        "readthedocs": "",
+        "helpdesk": "Perseo",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/perseo-core",
+            "telefonicaid/perseo-fe"
+        ],
+        "docker": []
```

Comment:
check fiware/perseo*

Reply:
Fixed in https://github.com/flopezag/fiware-tsc-dashboard/pull/38/commits/55ea9ac2cb5f6bb623a5b2897a9d1e5dede5c585